fluentnamer/src/main/java/com/azure/autorest/fluent/transformer/OperationNameNormalization.java

```diff
@@ -103,7 +104,7 @@ private static void rename(OperationGroup operationGroup, Map<String, String> re
 
         for (Operation operation : operationGroup.getOperations()) {
             String path = operation.getRequests().iterator().next().getProtocol().getHttp().getPath();
-            path = StringUtils.strip(path.trim(), "/");
+            path = TRIM_LEADING_AND_TRAILING_FORWARD_SLASH.matcher(path.trim()).group(1);
```

Comment:
This one seems causing `java.lang.IllegalStateException: No match found`.

Reply:
I'll guard this with a matches or make the regex more relaxed

Reply:
Decided to make the regex more relaxed

Reply:
@weidongxu-microsoft, @srnagar would it be possible to have AutoRest fail if there are errors during generation for the fluent-tests? It would have made the CI error much clearer

Reply:
I will try add `if %errorlevel% neq 0 exit /b %errorlevel%` to bat.

Reply:
https://github.com/Azure/autorest.java/pull/1298